test/end-to-end/pages/applicant/email.js

```diff
@@ -1,11 +1,16 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
 const pageUnderTest = require('app/steps/ui/applicant/email/index');
 
-module.exports = function (email) {
+async function enterApplicantEmail(language, email) {
+    const commonContent = language === 'en' ? commonContentEn : commonContentCy;
     const I = this;
-    I.seeCurrentUrlEquals(pageUnderTest.getUrl());
+
+    I.seeInCurrentUrl(pageUnderTest.getUrl());
     I.fillField('#email', email);
```

Comment:
I would add awaits here, always think good practice with a browser component to wait for code to finish before continuing to next action